Duplicati/Library/Main/Database/LocalDatabase.cs

```diff
@@ -135,7 +135,7 @@ private LocalDatabase(System.Data.IDbConnection connection)
             m_removeremotevolumeCommand.CommandText = @"DELETE FROM ""Remotevolume"" WHERE ""Name"" = ? AND (""DeleteGraceTime"" < ? OR ""State"" != ?)";
             m_removeremotevolumeCommand.AddParameters(3);
 
-            m_removedeletedremotevolumeCommand.CommandText = $@"DELETE FROM ""Remotevolume"" WHERE ""DeleteGraceTime"" < ? AND ""State"" == ""{RemoteVolumeState.Deleted.ToString()}""";
+            m_removedeletedremotevolumeCommand.CommandText = $@"DELETE FROM ""Remotevolume"" WHERE ""State"" == ""{RemoteVolumeState.Deleted.ToString()}"" AND ""DeleteGraceTime"" < ? OR LENGTH(""DeleteGraceTime"") > 12 "; // >12 is to handle removal of old records that were in milliseconds

```

Comment:
Was the old format actually in ticks (not milliseconds)?

Reply:
I think you're right. But the code doesn't change. Just the comment.

Reply:
Also, I think `AND` has precedence over `OR`.  Does that mean that the condition here is equivalent to
```
("State" == "Deleted" AND "DeleteGraceTime" < ?) OR LENGTH("DeleteGraceTime") > 12
```
Should it be instead
```
"State" == "Deleted" AND ("DeleteGraceTime" < ? OR LENGTH("DeleteGraceTime") > 12)
```

Reply:
I prefer explicit parenthesis, that makes it easier to understand the intent.